src/pages/discord/index.tsx

```diff
@@ -8,8 +8,8 @@ const Discord: FunctionComponent<LoginRequiredParams> = ({loginRequired}) => {
   return (
     <LoginRequired loginRequired={loginRequired}>
       {process.env.NEXT_PUBLIC_DISCORD_AUTHORIZE && (
-        <div className="w-1/2 flex space-y-6   flex-col">
-          <p className="prose lg:prose-xl">
+        <div className="grid place-items-center h-96">
+          <p className="prose lg:prose-xl text-white text-center">
```

Comment:
Changed to `grid` from `flex` to better center the items on the page, changed the text to white so it was easier to read.